azure-pipelines.yml

```diff
@@ -89,4 +89,34 @@ jobs:
             SERVICE: 'Anonymizer'
             WORKING_FOLDER: 'presidio-anonymizer'
 
-        
\ No newline at end of file
+    - job: FunctionalTests
+      dependsOn: 'SecurityAnalysis'
+      pool:
+        vmImage: 'ubuntu-16.04'
+
+      steps:
+        - task: DockerCompose@0
+          displayName: Start Presidio Cluster
+          inputs:
+            action: Run services
+            dockerComposeFile: docker-compose.yml
+            buildImages: true
+
+        - task: UsePythonVersion@0
+          inputs:
+            versionSpec: '3.8'
+
+        - script: |
+            python -m venv env
+            source env/bin/activate
+            python -m pip install --upgrade pip
+            pip install -r requirements.txt
+            pip install pytest-azurepipelines
+          workingDirectory: functional-tests
+          displayName: Install requirements
+
+        - script: |
+            source env/bin/activate
+            pytest

```

Comment:
Does this also output test results to ado?

Reply:
Yes, checked rhat